charts/crunchy/values.yaml

```diff
@@ -3,7 +3,8 @@ global:
     dbName: app #test
 crunchy: # enable it for TEST and PROD, for PR based pipelines simply use single postgres
   enabled: true
-  crunchyImage: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres-gis:ubi8-16.2-3.4-0
+  # it is recommended by platform team , not to use explicit images
+  #crunchyImage: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres-gis:ubi8-16.2-3.4-0
```

Comment:
Shouldn't we remove this to prevent anyone from ignoring the no explicit image recommendation?

Reply:
agreed, lets remove it completely